powerhome/playbook
# Fix rich text toolbar background

#### Screens

<img width="1002" alt="Screen Shot 2021-04-07 at 9 42 33 AM" src="https://user-images.githubusercontent.com/9784112/113876307-9c150580-9785-11eb-8c18-f96c40e1e2b1.png">


#### Breaking Changes

No

#### Runway Ticket URL

https://nitro.powerhrg.com/runway/backlog_items/NUXE-586

#### How to test this

Go to the rich text editor sticky variant and scroll to check for a solid not transparent background on the button toolbar

#### Checklist:


*The normal release cut off deadline is 3p EDT each week. Please reach out to the release team if you have an urgent request or need a release off cycle.*
